docs/resources/rule_mode.md

```diff
@@ -37,53 +37,53 @@ resource "wallarm_rule_mode" "tiredful_api_mode" {
 
 ## Argument Reference
 
-* `mode` - (Required) WAF node mode. Can be: `off`, `block`, `monitoring`, `default`. Aids to enable block mode granularly or turn off the WAF node for certain request parts.
-* `client_id` - (Optional) ID of the client to apply the rules to. The value is required for multi-tenant scenarios.
-* `action` - (Optional) Rule conditions. Possible attributes are described below.
+* `mode` - (**required**) Wallarm node mode. Can be: `off`, `block`, `monitoring`, `default`. Aids to enable block mode granularly or turn off the Wallarm node for certain request parts.
```

Comment:
safe blocking?

Reply:
and what is default? please explain here

Reply:
There's no default, this attribute is required. And only the listed modes are currently supported by the provider, safe_blocking isn't.